subsys/ipc/ipc_service/lib/icmsg.c

```diff
@@ -216,7 +216,9 @@ static void mbox_callback_process(struct k_work *item)
 	} else {
 		__ASSERT_NO_MSG(state == ICMSG_STATE_BUSY);
 
-		bool endpoint_invalid = (len != sizeof(magic) || memcmp(magic, rx_buffer, len));
+		/* Allow magic number longer than sizeof(magic) for future protocol version. */
```

Comment:
Cannot we add just like 2-3 characters for the version information right now?

Reply:
I assumed no dedicated numbers now can also mean there is a version - with no numbers.